react/controls/VolumeControl.tsx

```diff
@@ -0,0 +1,73 @@
+import React, { useState, FunctionComponent } from 'react'
+import {
+  IconVolumeOff as DefaultIconVolumeOff,
+  IconVolumeOn as DefaultIconVolumeOn,
+} from 'vtex.store-icons'
+
+import styles from '../styles/styles.css'
+
+export interface VolumeControlProps {
+  setVolume: (volume: number) => void
+  toggleMute: React.MouseEventHandler<HTMLButtonElement>
+  isMuted: boolean | null
+  cssHandles: Record<
+    'volumeContainer' | 'volumeButton' | 'volumeSlider',
+    string
+  >
+  volume: number | undefined
+  IconVolumeOn?: React.FC<unknown>
+  IconVolumeOff?: React.FC<unknown>
```

Comment:
Import the named exports instead of the whole `React` bag: `MouseEventHandler, `FC`